packages/fxa-profile-server/docs/API.md

```diff
@@ -68,6 +68,10 @@ The currently-defined error responses are:
 - [GET /v1/ecosystem_anon_id][ecosystem_anon_id]
 - [POST /v1/ecosystem_anon_id][ecosystem_anon_id-post]
 - [DELETE /v1/cache/:uid][cache_delete]
+- [GET /v1/\_core_profile][hidden from extra callers by default]
```

Comment:
@bbangert  Whenever I am saving this file this additional \ occurs due to no reason and even if we save without formatting this problem is going to arise due to `husky pre commit `